libs/application/templates/criminal-record/src/shared/constants.ts

```diff
@@ -0,0 +1,6 @@
+export enum ApiActions {
+  createApplication = 'createApplication',
```

Comment:
Same comment as above, I cannot find this in the `criminal-record-submission.service.ts`. Am I missing something?